content/sensu-go/5.19/reference/backend.md

```diff
@@ -1263,7 +1267,15 @@ Here are some log rotate sample configurations:
 }
 {{< /highlight >}}
 
-#### sysvinit
+Without the `postrotate` script, the backend will not reload.
+This will cause sensu-backend (and sensu-agent, if translated for the Sensu agent) to no longer write to the log file, even if logrotate recreates the log file.
```

Comment:
Per [Eric's comment here](https://github.com/sensu/sensu-enterprise-go/issues/1010), I think we need to make some adjustments or add a call out. 

Using `systemctl reload` sends a SIGHUP to the sensu-backend process.

Prior to 5.19.3, a reload will cause sensu-backend to fully restart -- so operators should be mindful of this so that they don't simultaneously restart all the members of a clustered backend. 

For version 5.19.3 and later, SIGHUP will cause only event logging components to reload. For backends running 5.19.3, it should be safe for all members of a backend cluster to receive a SIGHUP simultaneously.

Reply:
@cwjohnston I think I'm confused because this comment only mentions `systemctl reload`, which is in our systemd example above but is not in our sysvinit example here.

The postrotate script in our sysvinit example does not contain `systemctl reload` or even mention `reload` at all:
`kill -HUP `cat /var/run/sensu/sensu-backend.pid 2> /dev/null` 2> /dev/null || true`

Does this sysvinit script result in the same reload behavior you describe for `systemctl reload`?